src/libstd/fs.rs

```diff
@@ -81,9 +81,18 @@ use time::SystemTime;
 /// # }
 /// ```
 ///
+/// Note that, although read and write methods require a `&mut File`, because
+/// of the interfaces for [`Read`] and [`Write`], it is still possible to
+/// modify a file through a `&File`, either through methods that take `&File`
+/// or by retrieving a raw OS filehandle and modifying the file that way.
```

Comment:
"file handle"

Reply:
Or "underlying OS object", since on Linux it'd be a file *descriptor*.